From 8bc23cb14e5fabd8a58780ef7180b662f381e8a6 Mon Sep 17 00:00:00 2001
From: tribetmen <dima.koloskov.1997@yandex.ru>
Date: Sun, 1 Oct 2017 19:55:34 +0300
Subject: [PATCH] Fix InCall

Corrects an error: "InCall  : InCallPresenter - InCallActivity is null. Can't set requested orientation."
---
 InCallUI/src/com/android/incallui/InCallPresenter.java | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/InCallUI/src/com/android/incallui/InCallPresenter.java b/InCallUI/src/com/android/incallui/InCallPresenter.java
index fb96a1aa1..f4d4c515e 100644
--- a/InCallUI/src/com/android/incallui/InCallPresenter.java
+++ b/InCallUI/src/com/android/incallui/InCallPresenter.java
@@ -1876,15 +1876,7 @@ public void onDeviceOrientationChange(int orientation) {
      * @return returns whether the new orientation mode was set successfully or not.
      */
     public boolean setInCallAllowsOrientationChange(int orientation) {
-        if (mInCallActivity == null) {
-            Log.e(this, "InCallActivity is null. Can't set requested orientation.");
-            return false;
-        }
-
-        mInCallActivity.setRequestedOrientation(orientation);
-        mInCallActivity.enableInCallOrientationEventListener(
-                orientation == InCallOrientationEventListener.FULL_SENSOR_SCREEN_ORIENTATION);
-        return true;
+        return false;
     }
 
     /* returns TRUE if screen is turned ON else false */